src/Components/Board.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {SquareProps} from '../Types/TypeProps'
+import Square from "./Square";
+
+/**
+ * Handle board's cells display logic
+ * @param props 
+ * @returns 
+ */
+const Board: React.FC<SquareProps> = (props) => {
+
+  // Number of cells to display on the game board
+  const numberOfCells = 9;
+
+  // Group of numbers that identify each winning cell
+  const position = props.position;
+
+  /**
+   * Create cell template and properties
+   * @param i 
+   * @returns 
+   */
+  const renderSquare = (i: number) => {
```

Comment:
your relying on external state here which isn't good. You could run renderSquare with the exact same input, but get completely different outputs based of external factors. It makes more since just to send all the props to Square, and let Square handle all of this logic instead of relying on closure here.